src/main/java/com/alexx/employeesAndDepartments/controllers/DepartmentController.java

```diff
@@ -40,7 +46,10 @@ public ResponseEntity<Void> deleteDepartment(@PathVariable("departmentId")
                                                      @Min(value = 1, message = "ID of department must be positive.")
                                                      @Parameter(description = "ID of the existing department to remove")
                                                      Integer id) {
+        log.info("Delete department...");
+        log.debug("ID = {}", id);
```

Comment:
Imagine you see this message in logs. The same message is written in editDepartnemt. log.debug("ID = {}", id). Withoud additional information this message is useless. Please correct it

Reply:
In log I will see something like this:

if ok:
2023-05-28 12:08:29,918 INFO Delete department...
2023-05-28 12:08:29,918 DEBUG ID = 1
2023-05-28 12:08:30,144 INFO Department with ID = 1 deleted

or if not:
2023-05-28 12:08:24,411 INFO Delete department...
2023-05-28 12:08:24,411 DEBUG ID = 2
2023-05-28 12:08:24,997 WARN Department with this ID does not exist

What is incorrect? 
Should I write in log all the DTO?

Reply:
You should write more information. If there are mupltiple requests at the same time if will be hard to differentiate them, so you need to write more information